src/main/java/com/minecolonies/coremod/event/ColonyTagCapabilityProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.minecolonies.coremod.event;
+
+import com.minecolonies.api.colony.IColonyTagCapability;
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.world.chunk.Chunk;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.ICapabilitySerializable;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import static com.minecolonies.coremod.MineColonies.CLOSE_COLONY_CAP;
+
+public class ColonyTagCapabilityProvider implements ICapabilitySerializable<NBTBase>
```

Comment:
Rename this to MinecoloniesChunkCapabilityProvider.
This way we can reuse it incase we add caps later on to the chunk.